build-system/compile/closure-compile.d.ts

```diff
@@ -0,0 +1,2 @@
+interface Stream { }
+interface Writable { }
```

Comment:
Lol hmm. Would adding @ types/node help?

Reply:
I don't know why I didn't consider that, but yes that should work.